flink-table/flink-table-common/src/main/java/org/apache/flink/table/catalog/TableDistribution.java

```diff
@@ -120,7 +120,7 @@ private String asSerializableString() {
         if (getBucketKeys().isEmpty()
                 && getBucketCount().isPresent()
                 && getBucketCount().get() != 0) {
-            return "DISTRIBUTED INTO " + getBucketCount().get() + " BUCKETS\n";
+            return "DISTRIBUTED INTO " + getBucketCount().get() + " BUCKETS";
```

Comment:
there was a mess with `\n` in different places for `distributed`
as a result: depending on whether it is used with `partition by` or not there might be extra empty lines...

fix it and there is a test for that